src/components/buttons/yes-button.ts

```diff
@@ -1,41 +1,53 @@
-import { YES_BUTTON } from "@constants";
-
+import { YES_BTN_IMG } from '@constants';
```

Comment:
lets use double quotes just to be consistent for imports

Reply:
Changes applied, thank you.